src/schema/nodes/box_element.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * © 2019 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NodeSpec } from 'prosemirror-model'
+
+import { ManuscriptNode } from '../types'
+
+interface Attrs {
+  id: string
+  label: string | null
+}
+
+export interface BoxElementNode extends ManuscriptNode {
+  attrs: Attrs
+}
+
+export const box_element: NodeSpec = {
+  content: 'section_label? figcaption? section?',
```

Comment:
Shouldn't this be in the attrs?

Reply:
Since this is treated as a section and the label will already handled by the current section_label, this will minimize the changes, and I removed the label from the schema attributes.